Makefile

```diff
@@ -82,6 +87,11 @@ endif
 
 ifeq ($(OS),NetBSD)
 LOCAL_LDLIBS += -lutil
+
+ifdef USE_ONIGURUMA
```

Comment:
Why is this duplicated in the Linux and NetBSD blocks?  It should just be done once, outside of both.